web/views/tracker_view.ex

```diff
@@ -112,7 +112,7 @@ defmodule Healthlocker.TrackerView do
                   |> Map.get(:hours_slept)
       List.insert_at(list, 0, hours_slept)
     else
-      List.insert_at(list, 0, 0)
+      List.insert_at(list, 0, "null")
```

Comment:
not a "_fan_" of having "null" as a string. please _clarify_ the _reason_ for this. thanks!